scripts/posting_script/posts.py

```diff
@@ -61,6 +76,22 @@ def convert_posts():
             }
             dbs.cyberway_db['posttable'].save(message)
 
+            date_close = datetime.strptime("2106-02-07T06:28:15", '%Y-%m-%dT%H:%M:%S')
+            if (doc["cashout_time"] != date_close):
+                delay_trx = {
+                    "trx_id": "",
+                    "sender": doc["author"],
+                    "sender_id": dbs.convert_hash(doc["permlink"]) << 64 | doc["author"],
+                    "delay_until" : doc["cashout_time"], 
+                    "expiration" : doc["cashout_time"], 
```

Comment:
I suppose the expiration date should be later than the transaction execution date (for at least a few minutes)